deny.toml

```diff
@@ -37,6 +37,7 @@ allow = [
     "MPL-2.0",
     "BSD-3-Clause",
 ]
+private = {ignore = true}
```

Comment:
Please add this in the commit that made the originating change, and add spaces within `{}` to match surrounding style.